README.md

```diff
@@ -1,13 +1,60 @@
+<div align="center">
 
-Bowling Challenge
-=================
+<img src="images/logo.png" width="200px">
 
+![](https://img.shields.io/github/last-commit/arthurfincham/bowling-challenge)
+![](https://img.shields.io/github/languages/count/arthurfincham/bowling-challenge)
+![](https://img.shields.io/github/languages/code-size/arthurfincham/bowling-challenge)
+
+
+<h2>Preview</h2>
+
+<img src="images/preview.gif" width="400px">
+
+_command line interface for the game_
+
+</div>
+
+## Install
```

Comment:
This is a high quality readme. Was very easy for me to set up the project and test it out myself.